framework/src/meshgenerators/RenameBlockGenerator.C

```diff
@@ -99,7 +99,9 @@ RenameBlockGenerator::RenameBlockGenerator(const InputParameters & parameters)
   }
   else
   {
-    _old_block = getParam<std::vector<SubdomainName>>("old_block_name");
+    _old_block = isParamValid("old_block_name")
```

Comment:
use default

Reply:
Can not use default here because it will break line 81 condition and @dschwen want to avoid use `isParamSetByUser`

Reply:
nevermind it's because it s deprecated